compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -290,7 +431,32 @@ struct QuantizeBias final : public luci::CircleNodeMutableVisitor<bool>
 
     if (granularity == QuantizationGranularity::ChannelWise)
     {
-      // will be implemented.
+      assert(input->quantparam()->scale.size() == 1); // input scale's layer-wise
+      auto input_scale = input->quantparam()->scale[0];
+
+      assert(weight->quantparam() != nullptr); // weight scale's channel-wise
+      auto weight_scale = weight->quantparam()->scale;
+
+      auto circle_const = loco::must_cast<luci::CircleConst *>(node);
+
+      uint32_t size = circle_const->size<loco::DataType::FLOAT32>();
```

Comment:
The number of channels is the same with the size of bias (am I right?). If so, can you add an assert here?
```
assert(size == weight_scale.size());
```

Reply:
I will apply it